src/backend/joanie/core/filters/admin.py

```diff
@@ -0,0 +1,90 @@
+"""
+Admin API Resource Filters
+"""
+from typing import List
+
+from django.db.models import Q
+
+from django_filters import rest_framework as filters
+
+from joanie.core import models
+
+
+class OrganizationAdminViewSetFilter(filters.FilterSet):
+    """
+    OrganizationAdminFilter allows to filter this resource with a search for code and title.
+    """
+
+    search = filters.CharFilter(method="filter_search")
+
+    def filter_search(self, queryset, _name, value):
+        """
+        construct the full expression for search query param.
+        """
+        return queryset.filter(
+            Q(code__icontains=value) | Q(translations__title__icontains=value)
+        )
+
+    class Meta:
+        model = models.Organization
+        fields: List[str] = ["search"]
+
+
+class CourseRunAdminViewSetFilter(filters.FilterSet):
+    """
+    CourseRunAdminFilter allows to filter this resource with a search title.
+    """
+
+    search = filters.CharFilter(method="filter_search")
+
+    def filter_search(self, queryset, _name, value):
+        """
+        construct the full expression for search query param.
+        """
+        return queryset.filter(translations__title__icontains=value)
+
+    class Meta:
+        model = models.CourseRun
+        fields: List[str] = ["search"]
```

Comment:
I wonder if it could be interesting to be able to search within resource_link 🤔

Reply:
Mmmm, yes why not, but, with independent query param i think

Reply:
Yup !